website/docs/cli/cloud/migrating.mdx

```diff
@@ -82,3 +82,4 @@ terraform {
    }
  }
 ```
+~> **Warning:** Because the `cloud` block does not support the `prefix` argument, once you migrate you will need to refer to workspaces by their full name when using the Terraform CLI. For example, rather than `terraform workspace select prod`, you would need to run the command `terraform workspace select my-app-prod`.
```

Comment:
```suggestion
~> **Warning:** Because the `cloud` block does not support the `prefix` argument, once you migrate, you must refer to workspaces by their full name when using the Terraform CLI. For example, rather than `terraform workspace select prod`, you must run the command `terraform workspace select my-app-prod`.
```

Suggest changing "you will need" to "you must" to avoid future tense style issues and to use fewer words.